packages/spruce-skill-server/config/default.ts

```diff
@@ -0,0 +1,288 @@
+/* eslint-disable @typescript-eslint/camelcase */
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+// https://github.com/lorenwest/node-config/wiki/Configuration-Files
+import fs from 'fs'
+import baseErrors from './errors'
+
+export default function SpruceConfig<
+	aclType,
```

Comment:
Since these are generated by doing a `typeof` are they interfaces or types? Should we capitalize on them? We have historically capitalized types (thinking flow), not sure if we are carrying that forward.

Reply:
Yep, makes sense